charts/helm-chart/kubernetes-dashboard/values.yaml

```diff
@@ -85,7 +85,7 @@ app:
     # https://localhost:8443
     - localhost
     # - kubernetes.dashboard.domain.com
-    ingressClassName: internal-nginx
+    ingressClassName: ""
```

Comment:
If `ingressClassName` is ommited, , a [default Ingress class](https://kubernetes.io/docs/concepts/services-networking/ingress/#default-ingress-class) should be defined.

Reply:
I confirmed that it can be applied normally even if the value of `ingressClassName:` is omitted in the YAML file as follows.

```
apiVersion: networking.k8s.io/v1
kind: Ingress
metadata:
  name: example
spec:
  ingressClassName:
  rules:
    - host: example.com
      http:
        paths:
          - pathType: ImplementationSpecific
            backend:
              service:
                name: release-name-kubernetes-dashboard-web
                port:
                  name: web
```

But I've rarely seen values left blank in Kubernetes manifest or Helm Chart values yamls.